### PR TITLE
Feature/lessons function and display

### PR DESCRIPTION
**Pull request by sun-delletran**
[Commands]
- ## **On terminal:** 
    ### **Run pipenv**
        *   change directory  to ``` ph-sels-django-rodel/ ``` 
        *   run virtual environment ``` pipenv shell ``` 
        
    ### **Package installation** 
        *   change directory  to ``` ph-sels-django-rodel/ src/sels_fe``` 
        *  ``` npm install` 

    ### **Runserver**
        * react : on the current directory, ``` ph-sels-django-rodel/src/sels_fe ``` , run  ``` npm start  ``` 
        * backend : on a new, terminal change directory to ```ph-sels-django-rodel/src ``` 
        * and run ``` python manage.py runserver ``` 
	
- ## **On browser:**
    ### Check if the Server is running: Go to -
         ``` localhost:3000 ``` 

[Pre-conditions] 

	
[Expected Output] 
- ### At ``` localhost:3000/lesson``` 
        *  Go to ```localhost:3000/lesson```, Categories of the quizzes you may take will be displayed.
        *  Click ```Start Lesson``` of selected Category.
        *  You Should be able to access the questions, select answers, and view the result once done answering all of the questions.

[Notes] 
    ### In order to check for state and other information about followed users: Open Redux DevTools
        *  right click on the browser and open ```  Redux DevTools ```

### CODE REVIEW

File: src/answers_choices/models.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 class Question(models.Model):
     question = models.CharField(max_length=120)
     quiz = models.ForeignKey(QUIZ, default=1, on_delete=models.CASCADE)
-    description = models.CharField(max_length=255, blank=True, null=True)
+    description = models.CharField(max_length=1028, blank=True, null=True)
```
Comment (inahsanjuan) [inline src/answers_choices/models.py]: Use `TextField` instead of `CharField`

File: src/sels_fe/src/components/header/ESLNavBar.js [hunk 2]
```diff
@@ -15,34 +15,23 @@ import { AppBar, Toolbar, Typography, Stack, Divider } from "@mui/material"
...
   const isAuthenticated = useSelector((state) => state.Signin.isAuthenticated)
   const authUser = useSelector((state) => state.AuthUser.data)
-  const [anchorEl, setAnchorEl] = useState(null)
-  const [userMenuOpen, setUserMenuOpen] = useState(false)
   const history = useHistory()
 
-  const handleUserMenu = (event) => {
-    setAnchorEl(event.currentTarget)
-  }
+  // const handleUserMenu = (event) => {}
```
Comment (inahsanjuan) [inline src/sels_fe/src/components/header/ESLNavBar.js]: Delete unwanted code

File: src/sels_fe/src/components/header/ESLNavBar.js [hunk 3]
```diff
@@ -15,34 +15,23 @@ import { AppBar, Toolbar, Typography, Stack, Divider } from "@mui/material"
 import Popup from "reactjs-popup"
 import "reactjs-popup/dist/index.css"
 
+// import { Settings } from "@material-ui/icons"
```
Comment (inahsanjuan) [inline src/sels_fe/src/components/header/ESLNavBar.js]: Remove

File: src/sels_fe/src/components/header/ESLNavBar.js [hunk 4]
```diff
@@ -57,48 +46,57 @@ const ESLNavBar = () => {
...
                 </MenuItem>
+                {/* <Divider />
+                <MenuItem>
+                  <ListItemIcon>
+                    <Settings fontSize="Large" />
+                  </ListItemIcon>
+                  <ListItemText>Settings</ListItemText>
+                </MenuItem>
+                <Divider />
+                <br /> */}
```
Comment (inahsanjuan) [inline src/sels_fe/src/components/header/ESLNavBar.js]: Remove unwanted code

**Review by inahsanjuan — CHANGES_REQUESTED** [review]

File: src/sels_fe/src/components/header/ESLNavBar.js (hunk 2 above)
Comment (sun-delletran) [reply src/sels_fe/src/components/header/ESLNavBar.js]: Done.

File: src/sels_fe/src/components/header/ESLNavBar.js (hunk 3 above)
Comment (sun-delletran) [reply src/sels_fe/src/components/header/ESLNavBar.js]: Done.

File: src/sels_fe/src/components/header/ESLNavBar.js (hunk 4 above)
Comment (sun-delletran) [reply src/sels_fe/src/components/header/ESLNavBar.js]: Done.

**Pull request merged**